torch/ao/ns/_numeric_suite_fx.py

```diff
@@ -753,7 +754,7 @@ def extend_logger_results_with_comparison(
 def prepare_n_shadows_model(
     model: torch.nn.Module,
     example_inputs: Any,
-    qconfig_mappings: List[QConfigMapping],
+    qconfig_mappings: Union[List[QConfigMapping], QConfigMultiMapping],
```

Comment:
can we just take the new object instead of old object or new object?

Reply:
I can do that, just didn't want to break BC unnecessarily when its easy enough to maintain

Reply:
No one is using it yet, so there is no expectation of BC.  We can change API as needed until we get some customers, in fact this is the time when it's easiest to change the API.